cli/provider/cmd.go

```
@@ -737,6 +740,11 @@ func (c *CmdConfigurator) ValidateFlags(ctx context.Context, cmd *cobra.Command)
 		}
 	case "record", "test", "rerecord":
 
+		if cmd.Name() == "rerecord" {
+			updateTestSet, _ := cmd.Flags().GetBool("amend-testset")
```

don't ignore the error handling here.